fdbclient/admin_client.go

```diff
@@ -318,13 +318,21 @@ func (client *cliAdminClient) GetExclusions() ([]fdbv1beta2.ProcessAddress, erro
 	lines := strings.Split(output, "\n")
 	exclusions := make([]fdbv1beta2.ProcessAddress, 0, len(lines))
 	for _, line := range lines {
+		line = strings.TrimSpace(line)
+		if len(strings.Split(line, " ")) != 1 {
```

Comment:
What is the reasoning behind this? Could you add a comment above that?

Reply:
Mainly I was trying to remove the introductory line that 
```There are currently x servers or localities being excluded from the database: ```
But on talking another look I realised that this will be skipped in the later part of the code as it will not match the criteria.